### PR TITLE
Update codebase

### PR DESCRIPTION
**Pull request by cstamas**
Changes:
* update to 2.0
* drop plexus container default, use SISU
* cleanup and sanitize code

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -33,14 +33,46 @@
...
+  <dependencies>
+    <dependency>
+      <groupId>org.codehaus.plexus</groupId>
+      <artifactId>plexus-utils</artifactId>
+      <version>3.3.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.codehaus.plexus</groupId>
+      <artifactId>plexus-cipher</artifactId>
+      <version>2.0-SNAPSHOT</version>
```
Comment (michael-o) [inline pom.xml:48]: This can only be merged when PC 2.0 is released.

**Review by slachiewicz — APPROVED** [review]

**Review by michael-o — COMMENTED** [review]
Looks fine, but does not comply with the general formatting of the remaining code.

**Comment by cstamas** [comment]
> Looks fine, but does not comply with the general formatting of the remaining code.

fixed

**Review by michael-o — APPROVED** [review]
LGTM

**Pull request merged**